src/config.js

```diff
@@ -155,26 +155,52 @@ const config = convict({
       default: false,
       env: 'NOTIFY_EMAIL_DOWNLOAD_CONFIRMATION'
     },
-    caseDelivery: {
-      templateId: /** @type {SchemaObj<string | null>} */ {
-        format: String,
-        default: null,
-        nullable: true,
-        env: 'NOTIFY_CASE_DELIVERY_TEMPLATE_ID'
+    tb: {
+      caseDelivery: {
+        templateId: /** @type {SchemaObj<string | null>} */ {
+          format: String,
+          default: null,
+          nullable: true,
+          env: 'NOTIFY_CASE_DELIVERY_TEMPLATE_ID'
```

Comment:
Shall we rename these to be prefixed with TB_ ?

Reply:
I didnt want to just yet - as increases risk of breaking the live service